### ISSUE TITLE
In Service Locations, Phone number type should be honored within the component

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18131]
## Description 
When an editor chooses a type of phone that isn't `phone`, specific props should be used for the FE display:

- If a phone number is type SMS it uses the `sms` prop
- If a phone number is type Fax it uses the `not-clickable` prop and does NOT use the `message-aria-describedby` prop
- If a phone number is type TTY it uses the `tty` prop

We had done this in the [Phone component ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16148) however, after Service Locations launched, this seems to have reverted. Examples:
- Fax numbers that are type Fax but are not using the correct component props:
  - [Minneapolis Optometry](https://www.va.gov/minneapolis-health-care/locations/minneapolis-va-medical-center/#optometry)
  - [Caribbean Medical Records](https://www.va.gov/caribbean-health-care/medical-records-office/)
- TTY and SMS are rarely if ever used so I don't have examples of those but I would think this logic has been reverted just because we did it at the same time

This needs to be done in:
- VAMC Facility Health Services
- VAMC Non-clinical Services
- VBA Facility Services

## Acceptance Criteria
- [ ] Phone numbers that are type SMS use the `sms` prop
- [ ] Phone numbers that are type Fax use the `not-clickable` prop and do NOT use the `message-aria-describedby` prop
- [ ] Phone numbers that are type TTY use the `tty` prop
- [ ] Requires accessibility review

**Comment by davidmpickett** [comment]
@jilladams I added this to sprint 4 since it was injected

**Comment by laflannery** [comment]
Confirmed on prod!
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/ac73dbf1-b24f-49a4-80a1-09c66bece673)
